engine/api/src/main/java/io/deephaven/engine/table/iterators/ColumnIterator.java

```diff
@@ -4,14 +4,18 @@
 package io.deephaven.engine.table.iterators;
 
 import io.deephaven.engine.primitive.iterator.CloseableIterator;
+import io.deephaven.util.SafeCloseable;
 
 /**
  * Iteration support for values supplied by a column.
  *
  * @apiNote ColumnIterators must be explicitly {@link #close() closed} or used until exhausted in order to avoid
  *          resource leaks.
  */
-public interface ColumnIterator<DATA_TYPE> extends CloseableIterator<DATA_TYPE> {
+public interface ColumnIterator<DATA_TYPE> extends CloseableIterator<DATA_TYPE>, SafeCloseable {
```

Comment:
Same Q; if we decide that `CloseableIterator` is _not_ idempotent, we could still make that guarantee here if we wanted to.